backend/Tests/Api/GraphQL/Import/BakerImportHandlerTests.cs

```diff
@@ -0,0 +1,95 @@
+using Application.Api.GraphQL.Import;
+using Application.Import;
+using ConcordiumSdk.NodeApi;
+using ConcordiumSdk.NodeApi.Types;
+using ConcordiumSdk.Types;
+using Dapper;
+using FluentAssertions;
+using Tests.TestUtilities;
+using Tests.TestUtilities.Builders.GraphQL;
+using Tests.TestUtilities.Stubs;
+using Xunit.Abstractions;
+
+[Collection("Baker Import Handler")]
+public class BakerImportHandlerTest : IClassFixture<DatabaseFixture>
+{
+    private GraphQlDbContextFactoryStub _dbContextFactory;
+    private BakerImportHandler _target;
+
+    public BakerImportHandlerTest(DatabaseFixture dbFixture, ITestOutputHelper outputHelper)
+    {
+        _dbContextFactory = new GraphQlDbContextFactoryStub(dbFixture.DatabaseSettings);
+        _target = new BakerImportHandler(_dbContextFactory, new NullMetrics());
+
+        using var connection = dbFixture.GetOpenConnection();
+        connection.Execute("TRUNCATE TABLE graphql_accounts");
+        connection.Execute("TRUNCATE TABLE graphql_bakers");
+        connection.Execute("TRUNCATE TABLE graphql_account_release_schedule");
+        connection.Execute("TRUNCATE TABLE graphql_account_statement_entries");
+    }
+
+    [Fact]
+    public async Task TestFirstBlockAfterPaydayBakerAddition()
```

Comment:
Please add a comment why this test exists. I'm afraid it is otherwise unclear why this specific scenario is being tested.